src/api/structure/testing-unit.ts

```diff
@@ -87,9 +87,9 @@ export default abstract class TestingUnit extends BaseUnit {
 
         assertType(is.string, 'page', 'The page URL', this.pageUrl);
 
-        assertUrl(this.pageUrl, 'page');
+        assertUrl(this.pageUrl as string, 'page');
 
-        this.pageUrl = resolvePageUrl(this.pageUrl, this.testFile.filename);
+        this.pageUrl = resolvePageUrl(this.pageUrl as string, this.testFile.filename);
 
```

Comment:
That'd bad if we have to cast it because `this.pageUrl` can be null. Can we do something about it?

Reply:
We can move `pageUrl` property initialization from `Fixture` class into `TestingUnit` class:

https://github.com/DevExpress/testcafe/blob/87e3223f9bda84a7c10adc89fcfbbfa0f9cb39ce/src/api/structure/fixture.ts#L29